server/src/main/java/org/elasticsearch/search/retriever/RankDocsRetrieverBuilder.java

```diff
@@ -113,17 +114,19 @@ public void extractToSearchSourceBuilder(SearchSourceBuilder searchSourceBuilder
                 rankQuery = new RankDocsQueryBuilder(
                     rankDocResults,
                     sources.stream().map(RetrieverBuilder::topDocsQuery).toArray(QueryBuilder[]::new),
-                    false
+                    false,
+                    Float.MIN_VALUE
                 );
             } else {
                 rankQuery = new RankDocsQueryBuilder(
                     rankDocResults,
                     sources.stream().map(RetrieverBuilder::explainQuery).toArray(QueryBuilder[]::new),
-                    false
+                    false,
+                    Float.MIN_VALUE
                 );
             }
         } else {
-            rankQuery = new RankDocsQueryBuilder(rankDocResults, null, false);
+            rankQuery = new RankDocsQueryBuilder(rankDocResults, null, false, Float.MIN_VALUE);
```

Comment:
This should use a default member that you create and reference as suggested earlier. Alternately consider a convenience constructor that sets the default.